macrosynergy/panel/granger_causality_test.py

```diff
@@ -40,13 +40,13 @@ def granger_causality_test(
 ) -> Dict[Any, Any]:
     """
     Run Granger Causality Test on a standardized quantamental dataframe. Since the
-    Granger Causality Test is a pairwise test, only two tickers can be specified. Specify
-    either `tickers` or `cids` & `xcats`. When specifying `cids` & `xcats`, the user may
-    input one `cid` and two `xcats`; or two `cids` and one `xcat` to yield two tickers.
-    The function forms the list of tickers from the `cids` and `xcats` such that the
-    order of the formed tickers is preserved. The order of the tickers is important as
-    the first ticker is the one that is tested to Granger cause the second ticker. The
-    function tests whether the first ticker Granger causes the second ticker.
+    Granger Causality Test is a pairwise test, only two tickers can be specified.
+    The user may specify two tickers directly using the `tickers` parameter, or specify
+    one or two cross-sections and one or two extended categories using the `cids` and
```

Comment:
I think the meaning has changed here.

From 

one `cid` and two `xcats`; or two `cids` and one `xcat`

to 

one or two `cid` and one or two `xcats`

It sounds like you could pass two cids and two xcats

Reply:
Is that how it should be?

Reply:
I agree, will change this to read better